src/MSBuildLocator/Microsoft.Build.Locator.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFramework>net46</TargetFramework>
+    <TargetFrameworks>net46;netstandard2.0</TargetFrameworks>
```

Comment:
Should this be `netcoreapp2.0` rather than `netstandard2.0`. It seems weird that a `netstandard2.0` version of this package would only locate MSBuild via .NET Core SDK -- especially since `netstandard2.0` is supposed to be compatible with `net461`. Making the package `netstandard2.0` signals support that it probably shouldn't.